src/App.css

```diff
@@ -401,7 +401,7 @@ body {
 	-webkit-transform-style: preserve-3d;
 	transform-style: preserve-3d;
   }
-  body.msb-x .msb {
```

Comment:
I didn't analyse these styles, and I'm still not sure why we need all these classes. but for sure we don't need a `body` here :)